test/io/CsvReaderTest.ooc

```diff
@@ -0,0 +1,43 @@
+use ooc-base
+use ooc-collections
+use ooc-unit
+use ooc-io
+
+CsvReaderTest: class extends Fixture {
+	init: func {
+		super("CsvReader")
+		this add("verify entries", func {
+			filename := Text new(c"test/io/input/3x3.csv", 21)
+			reader := CsvReader open(filename)
```

Comment:
Then we can expect a segfault, like with every other test in Kean. The input files should never be removed, and if they are, the tests should be updated accordingly.